cmd/kstone-controller/etcdcluster-controller/controller.go

```diff
@@ -119,4 +192,7 @@ func (c *EtcdClusterCommand) AddFlags(fs *pflag.FlagSet) {
 		"",
 		"The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.",
 	)
+	fs.StringVar(&c.id, "id", uuid.New().String(), "the holder identity name")
```

Comment:
Is it better to put hostname as prefix of uuid? Users can easily find out which node's controller holds the lock.
```go
hostname, err := os.Hostname()
id = hostname + "_" + uuid.New().String()
```
refer to: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-scheduler/app/options/options.go#L307-L312
@maudL1n @tangcong WDYT?

Reply:
@maudL1n